lib/active_model/serializer/adapter/json_api.rb

```diff
@@ -119,10 +119,26 @@ def relationships_for(serializer)
           Hash[serializer.associations.map { |association| [association.key, { data: relationship_value_for(association.serializer, association.options) }] }]
         end
 
+        def expand_includes(includes, associations)
```

Comment:
can you write some yardoc about what this method does, and some example inputs and outputs?